sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -455,3 +455,7 @@ ALTER TABLE `amendments_history` MODIFY `created_time` timestamp NULL COMMENT 'c
 #IfNotColumnTypeDefault batchcom msg_date_sent datetime NULL
 ALTER TABLE `batchcom` MODIFY `msg_date_sent` datetime NULL;
```

Comment:
1. Also should have something like:
```
UPDATE `batchcom` SET `msg_date_sent` = NULL WHERE `msg_date_sent` = '0000-00-00 00:00:00';
```
2. And will need to figure out a way to blank out sql-mode for above statement and then reverting sql-mode back after the above statement

Reply:
Either we could update the default function we made earlier or we can add all of them in a separate file and execute without SQL mode and then re-update the SQL modes.

Reply:
Moreover, we have to do that for past default fixes of the date, date time and timestamp

Reply:
hi @ShubhamBansal1997 ,
it will end up looking something like this:
```
#IfNotColumnTypeDefault batchcom msg_date_sent datetime NULL
ALTER TABLE `batchcom` MODIFY `msg_date_sent` datetime NULL;
set @currentSqlMode  = SELECT @@sql_mode;
SET sql_mode = '';
UPDATE `batchcom` SET `msg_date_sent` = NULL WHERE `msg_date_sent` = '0000-00-00 00:00:00';
SET sql_mode = @currentSqlMode ;
#If
--
Agree we should also do with the prior timestamp changes, however would need to test what is stored (unlikely going to be 0000-00-00 00:00:00 format) in those instances.



```

Reply:
Updated